ios/Capacitor/Capacitor/Plugins/HttpRequestHandler.swift

```diff
@@ -141,7 +141,9 @@ class HttpRequestHandler {
     }
 
     public static func request(_ call: CAPPluginCall, _ httpMethod: String?, _ config: InstanceConfiguration?) throws {
-        guard let urlString = call.getString("url")?.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) else { throw URLError(.badURL) }
+        var allowedCharacters = CharacterSet.init("%").union(.urlQueryAllowed)
```

Comment:
This was introduced here: https://github.com/ionic-team/capacitor/pull/6169 which will encode encoded params.